src/azure-cli/service_name.json

```diff
@@ -0,0 +1,688 @@
+[
+  {
+    "Command": "az account",
+    "Description": "Manage Azure subscription information.",
+    "AzureServiceName": "Azure core",
```

Comment:
This is not Azure core, but Azure CLI core, pls help to work with delora to fix the excel